ilastik_feature_selection/wrapper_feature_selection.py

```diff
@@ -45,7 +44,7 @@ def kfold_train_and_predict(X, Y, classifier, k=5, indices=None, features=None):
         if features is None:
             features = np.array(list(range(X.shape[1])))
         features = np.array(list(features))
-        kf = cross_validation.KFold(len(indices), n_folds=k)
+        kf = model_selection.KFold(len(indices), n_folds=k)
         accurs = []
         for train, test in kf:
```

Comment:
this should be something like `for train, test in kf.split(indices)`

Reply:
Yeah, thanks, I already switched to model_selection.KFold(n_splits=k)
http://scikit-learn.org/0.18/modules/generated/sklearn.cross_validation.KFold.html#sklearn.cross_validation.KFold
http://scikit-learn.org/0.20/modules/generated/sklearn.model_selection.KFold.html
Am I still missing something?

Reply:
yes, you cannot simply use the referenced line now `for train, test in kf`, but something like (and i am not sure, please check) `for train, test in kf.split(indices)`...

Reply:
yeah, exactly!